components/events/Event.jsx

```diff
@@ -0,0 +1,56 @@
+
+import A from '../Anchor';
+export default class Event extends React.Component{
+  render(){
+    const { post }= this.props;
+    const { urlApi }= this.props;
+    return(
+        <article
+            className="eventItem"
+            style={{ backgroundImage: `url(${urlApi}${post.feature_image})` }}>
+            <div className="opacity">
+                <div className="eventContent">
+                    <A href={`${urlApi}/${post.slug}`} target="_blank">
+                        {post.title}
+                    </A>
+                </div>
+            </div>
+            <style jsx>{`
+                .eventItem {
+                    width: 33%;
```

Comment:
More values will be needed for smaller devices, we should go with a mobile first strategy; I'm ok to start with this though.

Reply:
Yeaa Definitely... Let us know what should be your plan please.